### ISSUE TITLE
Remove unused node_modules.sums

### ISSUE THREAD

**Issue by weberhofer** [openSUSE/obs-service-node_modules#39]
I have started using this service in 
https://build.opensuse.org/package/show/devel:kubic/traefik

A node_modules.sums file is generated - should that included in the .spec, too?

**Comment by AdamMajer** [comment]
No, it's not used by anything. Actually, it's mostly redundant anyway since package-json.lock contains these checksums as well.

I will look and remove this file generation in the future.

**Comment by weberhofer** [comment]
Thanks for your response, I will remove that file in my package.
